packages/office-ui-fabric-react/src/components/Persona/PersonaPresence.tsx

```diff
@@ -17,15 +17,24 @@ export class PersonaPresence extends React.Component<IPersonaProps, {}> {
   }
 
   public render(): JSX.Element | null {
-    let { presence } = this.props;
+    let { presence, coinSize } = this.props;
+    let coinSizeWithPresenceIconStyle = coinSize ? { fontSize: coinSize / 6, lineHeight: coinSize / 3 + 'px' } : undefined;
```

Comment:
3 and 6 are magic numbers. Can you put them in consts with descriptive names to improve readability?